Sources/GoodExtensions/Foundation/OptionSetExtension.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  OptionSetExtension.swift
+//  GoodExtensions
+//
+//  Created by Filip Šašala on 14/02/2022.
+//  Copyright © 2022 GoodRequest. All rights reserved.
+//
+
+import Foundation
```

Comment:
@Plajdo treba tu ten Foundation alebe by slo aj bez neho?

Reply:
vyzera ze to funguje aj bez toho